app/server/index.js

```diff
@@ -154,6 +154,15 @@ app.prepare().then(() => {
   server.use(bodyParser.json());
   server.use(cors());
 
+  // Tell search + crawlers not to index non-production environments:
+  server.use(({res, next}) => {
+    if (process.env.NODE_ENV !== 'production') {
```

Comment:
`NODE_ENV` will be `production` for all namespaces, as defined in the `package.json`'s start script. `process.env.NAMESPACE` is what you want to look at here
```suggestion
    if (!process.env.NAMESPACE || !process.env.NAMESPACE.endsWith('-prod') {
```

Reply:
Thanks, I'd seen that and was going to ask about it!